esdm/esdm_drng_mgr.c

```diff
@@ -452,7 +459,7 @@ static uint32_t esdm_drng_seed_es_nolock(struct esdm_drng *drng, bool init_ops,
 		}
 
 		esdm_fill_seed_buffer(
-			&seedbuf, esdm_get_seed_entropy_osr(drng->fully_seeded),
+			&seedbuf, esdm_get_seed_entropy_osr(drng->was_fully_seeded_once),
```

Comment:
IMHO this is not right: when the DRBG gets unseeded, it must be fully reseeded again, no?